### PR TITLE
[52] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Onyinye101**
## Task: `[52] Develop a Contact Form Page using HTML`
- `created a contact form for an e-commerce website and included screenshots of the implementation`

Screenshot 2020-10-05 at 9 53 58 AM

### CODE REVIEW

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+<body>
+    <form action="" method="POST"> 
+        <fieldset>
+            <legend><h4>CONTACT US</h4></legend>
+            <!-- This is for the User's fullname-->
+            <label for="fname">Full Name:</label><br>
+            <input type="text" id="fullname" name="fullname"><br><br>
+
+            <!-- This is for the User's Email-->
+            <label for="fname">Email:</label><br>
```
Comment (ToluAkin) [inline submissions/onyinyeChukwuka/HTML-Forms/contactform.html:16]: update the label for the email @Onyinye101

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <label>Subject:</label><br>
+            <textarea rows= "2" cols = "50" name = "message"> </textarea><br><br>
+
+
+            <!-- This is for the message input -->
+            <label>Message:</label><br>
+            <textarea rows= "10" cols = "50" name = "message"> </textarea><br><br>
+
+            <!-- This is for the submit button to post the news-->
+            <input type="Submit" value="Submit">
```
Comment (ToluAkin) [inline submissions/onyinyeChukwuka/HTML-Forms/contactform.html:29]: @Onyinye101 input type should be in lowercase

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html (hunk 2 above)
Comment (Onyinye101) [reply submissions/onyinyeChukwuka/HTML-Forms/contactform.html]: I've made the corrections

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html (hunk 1 above)
Comment (Onyinye101) [reply submissions/onyinyeChukwuka/HTML-Forms/contactform.html]: I've made the corrections

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<html lang="en">
```
Comment (adaugochi) [inline submissions/onyinyeChukwuka/HTML-Forms/contactform.html:1]: Add the doctype

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/onyinyeChukwuka/HTML-Forms/index.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/onyinyeChukwuka/HTML-Forms/index.html:1]: This file is not meant to be on this task

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/onyinyeChukwuka/HTML-Forms/contactform.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CONTACT FORM</title>
+</head>
+<body>
+    <ul>
+        <li> <a href="#"> Home </a></li>
+        <li> <a href="#"> WOMEN</a></li>
+        <li> <a href="#"> MEN  </a></li>
+        <li> <a href="#"> Contact Form </a></li>
+        <li> <a href="index.html" target="_blank"> Product </a></li>
```
Comment (adaugochi) [inline submissions/onyinyeChukwuka/HTML-Forms/contactform.html:14]: Remove this link. You will do this in your next task

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**